### PR TITLE
Added gulp build to README

### PR DESCRIPTION
**Pull request by RajatArora08**
- [X] I've checked that this isn't a duplicate pull request.

Is it regarding a new swag proposal or something else?  Something else.
Fill the corresponding section below:

(New Swag Proposal)

* Company:
* Reference URL: 

Q. Have you received swag from this?  
Q. How long did it take?  
Q. Does it ship worldwide?  

(Something else)  
Please describe your changes here: 
Adds gulp build to README #117

### CODE REVIEW

File: site/README.md [hunk 1]
```diff
@@ -15,12 +15,16 @@ This folder contains the source code of the devSwag website (hosted at http://de
     ```
     npm install gulp-cli -g
     ```
-1. Open a terminal in the `swag-for-dev/site` directory. Type
+1. Open a terminal in the `swag-for-dev/site` directory. To install the dependencies, type 
     ```
     npm install
     ```
-    into the terminal to install the dependencies. To start the webserver, type
+    To build the webserver, type
```
Comment (aslafy-z) [inline site/README.md]: This does not build the webserver but the website.

File: site/README.md [hunk 2]
```diff
@@ -15,12 +15,16 @@ This folder contains the source code of the devSwag website (hosted at http://de
...
     ```
-    gulp
+    gulp build
```
Comment (aslafy-z) [inline site/README.md]: This build command is not needed. Calling only `gulp` will build the assets and serve it through a webserver. Calling `gulp build` will only build the assets, it does a production build.

**Comment by aslafy-z** [comment]
Also, I did some enhancements to this doc in #76 . Feel free to check and comment .

**Comment by RajatArora08** [comment]
@aslafy-z Thanks for the review! I guess the Docker is not yet merged on the repo hence I didn't see it.
Regarding this issue, I did a clean install of the repo, but `gulp` fails with the issue `Error: File not found with singular glob: ~/swag-for-dev/site/dist `. After your comment, I digged down into the issue and I believe the issue is because gulp default task runs in parallel and the webserver is starts before the files are built into the dist directory. I created another PR (https://github.com/swapagarwal/swag-for-dev/pull/121) for the fix that worked for me.

**Pull request closed without merging**